lib/blocks.php

```diff
@@ -48,6 +48,9 @@ function gutenberg_reregister_core_block_types() {
 				'post-tags.php'           => 'core/post-tags',
 				'site-title.php'          => 'core/site-title',
 				'template-part.php'       => 'core/template-part',
+				'query.php'               => 'core/query',
+				'query-loop.php'          => 'core/query-loop',
```

Comment:
Is there ever going to be a query without a loop? Could it be absorbed?

Reply:
What do you mean by absorbed? It needs to be movable against its pagination.

Reply:
I thought pagination would sit outside in most cases